Libraries/Text/RCTText.m

```diff
@@ -237,7 +237,11 @@ - (BOOL)canBecomeFirstResponder
 - (BOOL)canPerformAction:(SEL)action withSender:(id)sender
 {
   if (action == @selector(copy:)) {
```

Comment:
But, I believe, we have to delegate this decision to `[self.nextResponder canPerformAction: withSelector:]` instead of returning `NO`.